### PR TITLE
Navbar - Add right navbar on Figma #386

### PR DESCRIPTION
**Pull request by mktana**
# Description
adding navbar. it's first version.

## screenshots
Navbar-Databus

### CODE REVIEW

**Comment by shootermv** [comment]
why file suffixes is `jsx` and not `tsx`?

File: src/pages/Navbar/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { FireFilled } from '@ant-design/icons';
+
+const Logo = () => {
+  return (
+    <div className="logo">
+      <div className="logo-icon">
+        <FireFilled />
```
Comment (NoamGaash) [inline src/pages/Navbar/Logo.jsx:7]: can we use the same logo?

Comment (mktana) [reply src/pages/Navbar/Logo.jsx]: do you mean to change the name of the class?

Comment (NoamGaash) [reply src/pages/Navbar/Logo.jsx]: I mean this one
image

Comment (mktana) [reply src/pages/Navbar/Logo.jsx]: yes, I changed it, I need the logo image.

**Comment by NoamGaash** [comment]
Hi, how are you? do you need any help or have any question?

**Comment by NoamGaash** [comment]
seems like @mktana isn't here anymore
I'm closing the PR

@mktana  - feel free to re-open when you feel it's relevant

**Pull request closed without merging**